utils/config.go

```diff
@@ -387,6 +391,20 @@ func ValidateLdapFilter(cfg *model.Config) *model.AppError {
 	return nil
 }
 
+func ValidateLocales(cfg *model.Config) *model.AppError {
+	if len(*cfg.LocalizationSettings.AvailableLocales) > 0 {
+		for _, word := range strings.Split(*cfg.LocalizationSettings.AvailableLocales, ",") {
+			if word == *cfg.LocalizationSettings.DefaultClientLocale {
+				return nil
+			}
+		}
+
+		return model.NewLocAppError("ValidateLocales", "utils.config.validate_locale.app_error", nil, "")
+	}
+
+	return nil
```

Comment:
If there's no locales, we default to english? That's what this is doing right? Just checking